01_01_Basic_Rules.txt

```diff
@@ -413,34 +413,54 @@ The equations to calculate your bonuses to Saving Throws are as follows:
 Healing Injuries
 ==============================
 
-	Ouch! you've just been shot! You'd better call for a Doctor or apply a med 
-kit before you bleed out! "But how do I use a med kit? Or if I'm a Doctor, how 
-do I stabilize and heal an ally?" I'm glad you asked. The rules are as follows:
-The Medicine DC is equal to the damage dealt. On a failed skill check, no healing 
-occurs. On a success, heal an amount equal to Medicine skill + DC surplus.
-
-	For example, a Player Character is dealt 70 damage. Their Doctor has 25 
-skill in medicine. Medic rolls an 86 (after adding their Medicine Skill), so they 
-heal 25+(86-70) for 41 health healed.
-
-	Individual wounds are healed individually. 3x 20-damage wounds need three 
-rolls of 21 or higher before they are fully healed.
-
-	For example, a Player Character takes 70 damage. Medic has 25 skill. Medic 
-rolls an 86 (with Medicine Skill), so they heal 25+(86-70) for 41 Health. Now 
-the Character still has a 29 damage deficit (70-41), so the medic would need to 
-roll over a DC of 29 to heal the remaining 29 damage. (The Doctor Cannot overheal 
-a wound unless otherwise stated).
-
-	Doctors need not use a medic-kit so long as the have their medicine bag (or 
-equivalent) on their person. If they do not have their medicine bag with them 
-when they are trying to heal a teammate, they can make the attempt at a +20 DC.
-
-Using a medic-kit:
-
-	Medic-kit, non-medic user: works as if you are a medic with skill 30.
-
-	Medic-kit, medic user: Works as normal, but add 30 to amount healed.
+    The galaxy is a dangerous place, and you may find in the course of your 
+adventure that you need to heal a wound or even stabalize a friend who is on the
+doorstep of death. The first thing you'll need to know is that treating the 
+injuries of a cybernetic character usually requires the repair skill, while 
+treating a biological character requires the medicine skill. Items such as a 
+medkit or repair kit is required for most characters to be able to treat a wound
+or stabalize a dying character. For stabalizing a dying character, see the 
+"Stabalizing" section under "Death and Dying" in the Combat Rules.
+
+    Using a medkit or a repair kit to heal a wounded character takes 3 actions
```

Comment:
Order suggestion: Using a ... to heal a wounded character in combat takes 3 Actions.